tests/test_mutualauth.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import unittest
+
+import falcon
+from falcon_mutualauth.mutualauth import Authorize
+
+
+class FakeRequest(object):
+
+    def __init__(self, role):
+        self.__role = role
+        self.env = {'REMOTE_ADDR': 'http://test.test'}
+        self.user_agent = 'pytest'
+
+    def get_header(self, header):
+        assert header == 'X-User-Roles'
+        return self.__role
+
+
+class FakeResource(object):
+
+    def __init__(self, authorized_for):
+        self.authorized_for = authorized_for
+
+
+class TestAuthorize(unittest.TestCase):
```

Comment:
Could these tests be written similarly to https://github.com/RackSec/overcorps/blob/master/tests/functional_tests.py#L439-L484? The reason being, falcon-mutualauth is being used specifically with falcon -  as such it seems like exercising that logic might be helpful.